src/main/java/com/github/wovnio/wovnjava/Headers.java

```diff
@@ -208,6 +211,13 @@ String redirectLocation(String lang) {
         }
     }
 
+    /**
+     * @return String Returns request URL without any language code
+     */
+    String getUrlWithoutLanguageCode() {
```

Comment:
confusing...

Reply:
Looks good.

Reply:
I see.

Reply:
Alright, lets make it less confusing.

At first I named it `originalLocation`, as opposed to the `redirectLocation` function above it. But that felt imprecise. So I changed it to the current name because we use the function when we want to read the request URL, just stripped of any language code. (Similarly, the function above is used for when we want the request URL with the language code included.)

Do you have a suggestion for how we can name it better?

Reply:
First I expected `this.url` includes scheme, because [rfc says](https://tools.ietf.org/html/rfc3986).

Reply:
Yes, I think that the original author may have expected the same thing. And since the tests did not cover the request format, it resulted in this bug.